src/TwigComponent/tests/Unit/CVATest.php

```diff
@@ -0,0 +1,215 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\TwigComponent\Tests\Unit;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\UX\TwigComponent\CVA;
+
+/**
+ * @author Mathéo Daninos <matheo.daninos@gmail.com>
+ */
+class CVATest extends TestCase
+{
+    /**
+     * @dataProvider recipeProvider
+     */
+    public function testRecipes(array $recipe, array $recipes, string $expected): void
+    {
+        $recipeClass = CVA::init($recipe);
+
+        $this->assertEquals($expected, $recipeClass->resolve($recipes));
+    }
+
```

Comment:
Could you maybe add one of two basic tests to check Exceptions / errors / invalid inputs ?